packages/bridge/src/vl/base/BaseFilter.ts

```diff
@@ -1,4 +1,6 @@
-import mitt from 'mitt';
+import { Emitter } from 'mitt';
```

Comment:
I think we're importing this dependency twice :(
It would be better to use `import { Emitter }, mitt from 'mitt';` or any other statement but importing it only once